doc/contributor/updating-ruby.md

```diff
@@ -196,9 +196,7 @@ Update all of these:
 * Reset `truffleruby-abi-version.h` to `$RUBY_VERSION.1` and `lib/cext/ABI_check.txt` to `1` if `RUBY_VERSION` was updated.
   * use `$TRUFFLERUBY_VERSION.1` in `truffleruby-abi-version.h` instead when on a release branch.
 * Update `versions.json`
-  * with bundled gem versions provided by `cat ../ruby/gems/bundled_gems | sort` (ensure the target Ruby version is checked out in the `../ruby` directory),
-  * default gem versions provided by `ls -l lib/gems/specifications/default`
-  * and `gem` gem version provided by `grep 'VERSION =' lib/mri/rubygems.rb`
+  * run `RUBY_SRC_DIR=../ruby-$VERSION tool/update-gem-versions-list.rb`
```

Comment:
The env var is RUBY_SOURCE_DIR in the script

Reply:
Whoops. I renamed it to better match other doc changes and missed this reference. Thanks.